### ISSUE TITLE
Fastlogin has a problem with Authme email registration

### ISSUE THREAD

**Issue by Chaostarts** [games647/FastLogin#1267]
### What happened?

If i use this config in Authme

    registration:
        # Enable registration on the server?
        enabled: true
        # Send every X seconds a message to a player to
        # remind him that he has to login/register
        messageInterval: 5
        # Only registered and logged in players can play.
        # See restrictions for exceptions
        force: true
        # Type of registration: PASSWORD or EMAIL
        # PASSWORD = account is registered with a password supplied by the user;
        # EMAIL = password is generated and sent to the email provided by the user.
        # More info at https://github.com/AuthMe/AuthMeReloaded/wiki/Registration
        type: EMAIL 
        # Second argument the /register command should take: 
        # NONE = no 2nd argument
        # CONFIRMATION = must repeat first argument (pass or email)
        # EMAIL_OPTIONAL = for password register: 2nd argument can be empty or have email address
        # EMAIL_MANDATORY = for password register: 2nd argument MUST be an email address
        secondArg: NONE 

The premium player enter their email to register and rejoin the server .The premuim player will be AutoLogined by FastLogin so that they can bypass the email  authentication.

### Plugin list

AuthMe, AuthMeVelocity, Fastlogin ,  PlugManX, PowerBoard, ProtocolLib,  tpLogin, UTitleAuth, Vault, ViaBackwards, ViaRewind, ViaVersion

### Configuration file

https://hastebin.com/share/ihoregemon.yaml

### Plugin version

#1326

### Platform

Velocity

### Relevance

- [X] I tried the [latest build](https://ci.codemc.io/job/Games647/job/FastLogin/) 
(build refers to development builds not necessary a release version; i.e. v1.10 is out of date)

- [X] I checked for existing tickets -
If there are, please vote them with a thumbs reaction and not create new ones

**Comment by games647** [comment]
If you have `autoRegister` disabled, users should not be able to to enable auto logins for themselves or it doesn't event enable it automatically, because your auth plugins blocks the execution of commands until registered and logged in. Do you excluded the premium command for this restriction?

**Comment by Chaostarts** [comment]
> If you have `autoRegister` disabled, users should not be able to to enable auto logins for themselves or it doesn't event enable it automatically, because your auth plugins blocks the execution of commands until registered and logged in. Do you excluded the premium command for this restriction?

Authme uses an email initial password registration method. After entering the email address, an email with the initial password will be sent and an account will be registered. If you fail to log in after the set time, your account will be automatically unregistered. After obtaining the initial password, premuim players who rejoin again will be automatically logged by FastLogin.That means they can bypass the initial password to register an account

**Comment by games647** [comment]
Yes, but if you don't use `autoRegister` this premium login is opt-in. Then FastLogin doesn't activate it. It requires the use of the `/premium` command by the player itself.

**Comment by Chaostarts** [comment]
> Yes, but if you don't use `autoRegister` the premium login is opt-in. Then FastLogin doesn't activate it. It requires the use of the `/premium` command by the player itself.

emmm...I want players to be able to be autologined automatically rather than use /premium.

**Comment by Chaostarts** [comment]
https://github.com/HaHaWTH/AuthMeReReloaded/actions/runs/12275608449
Authmerereloaded added a email 2fa to solve this problem
